android/src/main/java/com/facebook/reactnative/androidsdk/FBSettingsModule.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.reactnative.androidsdk;
+
+import androidx.annotation.Nullable;
+
+import com.facebook.FacebookSdk;
+import com.facebook.react.bridge.BaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+
+public class FBSettingsModule extends BaseJavaModule {
```

Comment:
Can you add the @ReactModule / NAME const here like https://github.com/facebook/react-native-fbsdk/blob/master/android/src/main/java/com/facebook/reactnative/androidsdk/FBAccessTokenModule.java#L40. This is useful when using the new turbomodules infra.